### PR TITLE
chore: improve ListV2 cypress scripts

### PR DESCRIPTION
**Pull request by Tooluloope**
## Description
Improve Listv2 tests

Fixes # (issue)
https://theappsmith.slack.com/archives/C02JV8G1MP0/p1676925949886009

## Type of change

> Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

- Cypress

### Dev activity
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] PR is being merged under a feature flag

### QA activity:
- [ ] Test plan has been approved by relevant developers
- [ ] Test plan has been peer reviewed by QA
- [ ] Cypress test cases have been added and approved by either SDET or manual QA
- [ ] Organized project review call with relevant stakeholders after Round 1/2 of QA
- [ ] Added Test Plan Approved label after reveiwing all Cypress test

### CODE REVIEW

**Comment by Aishwarya-U-R** [comment]
Can we please also fix [ListV2_PageNo_PageSize_spec]([https://cloud.cypress.io/projects/eyxvp8/runs/30230/test-results?actions=%5B%5D&brows[…]2%2C%22label%22%3A%22FAILED%22%7D%5D&testingTypesEnum=%5B%5D](https://cloud.cypress.io/projects/eyxvp8/runs/30230/test-results?actions=%5B%5D&browsers=%5B%5D&groups=%5B%5D&isFlaky=%5B%5D&modificationDateRange=%7B%22startDate%22%3A%221970-01-01%22%2C%22endDate%22%3A%222038-01-19%22%7D&orderBy=EXECUTION_ORDER&oses=%5B%5D&specs=%5B%5D&statuses=%5B%7B%22value%22%3A%22FAILED%22%2C%22label%22%3A%22FAILED%22%7D%5D&testingTypesEnum=%5B%5D))

**Comment by Tooluloope** [comment]
@Aishwarya-U-R 

> Can we please also fix [ListV2_PageNo_PageSize_spec](https://cloud.cypress.io/projects/eyxvp8/runs/30230/test-results?actions=%5B%5D&browsers=%5B%5D&groups=%5B%5D&isFlaky=%5B%5D&modificationDateRange=%7B%22startDate%22%3A%221970-01-01%22%2C%22endDate%22%3A%222038-01-19%22%7D&orderBy=EXECUTION_ORDER&oses=%5B%5D&specs=%5B%7B%22label%22%3A%22Regression_TestSuite%2FClientSideTests%2FWidgets%2FListV2%2FListV2_PageNo_PageSize_spec.js%22%2C%22value%22%3A%22%5B%5C%22b6438fe7-9a2c-4738-9b3d-cfe2862df5f5%5C%22%5D%22%7D%5D&statuses=%5B%7B%22value%22%3A%22FAILED%22%2C%22label%22%3A%22FAILED%22%7D%5D&testingTypesEnum=%5B%5D)

This test runs fine both on Local and cypress.
From the Link you sent this issue is from the Query Page not from widget.
Screenshot 2023-02-23 at 09 54 56

File: app/client/cypress/integration/Regression_TestSuite/ClientSideTests/Widgets/ListV2/Listv2_BasicChildWidgetInteraction_spec.js [hunk 1]
```diff
@@ -130,7 +151,7 @@ describe("List widget v2 - Basic Child Widget Interaction", () => {
   it("4. Switch widget", () => {
     // Drop Select widget
     dragAndDropToWidget("switchwidget", "containerwidget", {
-      x: 50,
+      x: 250,
       y: 50,
     });
```
Comment (Aishwarya-U-R) [inline app/client/cypress/integration/Regression_TestSuite/ClientSideTests/Widgets/ListV2/Listv2_BasicChildWidgetInteraction_spec.js:156]: Can we please reduce the number of `'it'` functions in this spec? A test script should do some reasonable checks.

Comment (Tooluloope) [reply app/client/cypress/integration/Regression_TestSuite/ClientSideTests/Widgets/ListV2/Listv2_BasicChildWidgetInteraction_spec.js:156]: Can you explain further.

The `it` functions are the different widget interactions test cases

**Comment by Aishwarya-U-R** [comment]
Yes, lets check why `.t--form-control-SWITCH input` element is not found while running ListV2_PageNo_PageSize_spec.js & fix it.

**Review by Aishwarya-U-R — APPROVED** [review]

**Pull request merged**